README.md

```
@@ -1,61 +1,190 @@
+> wget -O assets/gifs/ubuntu-logo.png "https://upload.wikimedia.org/wikipedia/commons/a/ab/Logo-ubuntu_cof-orange-hex.svg"
+> wget -O assets/gifs/debian-logo.png "https://upload.wikimedia.org/wikipedia/commons/0/0d/Debian-logo.svg"
+> git add assets/gifs/*
+> git commit -m "Add README demo assets"
+> git push origin main
+> ```
+
+---
+
+## 📖 Table of Contents
+
+- [About](#about)  
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)